latest/Dockerfile.amd64

```diff
@@ -0,0 +1,8 @@
+FROM zenko/cloudserver:latest-7.4.5
```

Comment:
the old `.drone.yml` used to:
```
git clone https://github.com/scality/cloudserver.git scality-s3server
cd ./scality-s3server
git checkout 7.4.4.10
```
I guess:
- 7.4.5 is now the current version we want
- `zenko/cloudserver` is the "replacement" we want to use instead of `scality/cloudserver`
?

Reply:
`scality/cloudserver` is a github repo. If you look at the readme https://github.com/scality/cloudserver#docker `zenko/cloudserver` seems to be the "new" official docker hub repo, so I switched to this as a base image because building the image by our own just to replace the entrypoint script makes no sense for me. Sadly, there is no pre-build 7.4.4* image available but I would assume 7.4.5 will also work.